### PR TITLE
CUSTCOM-194 The delete-jvm-options command shouldn't require to specify the Java version prefix

### PR DESCRIPTION
**Pull request by MeroRai**
# Description
This is a bug fix to fix an issue where the `delete-jvm-options` command required the specify Java version prefix to execute successfully. 

# Testing

### New tests
None

### Testing Performed
Build test
Manually tested the `delete-jvm-options` command with and without the Java version prefix with the following commands: 
-  `./asadmin delete-jvm-options '[Azul-1.8.0u222|1.8.0u500]-XX\:+UseOpenJSSE'` 
-  `./asadmin delete-jvm-options '-XX\:+UseOpenJSSE'` 

### Testing Environment

# Documentation
Not required

### CODE REVIEW

**Comment by MeroRai** [comment]
Jenkins test please

File: nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/commands/DeleteJvmOptions.java [hunk 1]
```diff
@@ -158,12 +159,22 @@ public Object run(JavaConfig jc) throws PropertyVetoException, TransactionFailur
...
         SingleConfigCode<JvmOptionBag> scc = (JvmOptionBag bag1) -> {
             List<String> jvmopts = new ArrayList<>(bag1.getJvmRawOptions());
             int orig = jvmopts.size();
-            // using new JvmOption(option).toString() (instead op option directly) to make sure the correct formatting is applied.
-            boolean removed = jvmopts.removeIf(option -> toRemove.contains(new JvmOption(option).toString()));
+            boolean removed = false;
+            Iterator<String> iter = jvmopts.iterator();
+            for (String toRemoveOption : toRemoveOptions) {
+                String option = new JvmOption(toRemoveOption).option;
+                while (iter.hasNext()) {
```
Comment (rdebusscher) [inline nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/commands/DeleteJvmOptions.java]: This construct fails in some cases when multiple options are removed.  Iterator is only initialised once, so you can have the situation that the second `toRemoveOption` value is already in the part of the iterator which has been traversed already.

Comment (MeroRai) [reply nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/commands/DeleteJvmOptions.java]: @rdebusscher, you are right. It was a dumb mistake on my part

**Comment by MeroRai** [comment]
Jenkins test please

**Pull request merged**